.docker/docker_start_script.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+#Configure web server
+
+# cd /usr/src/app/e-mission-server
```

Comment:
why comment this out? 

WORKDIR is set to `/usr/src/app` so without this `cd`, it is not clear that we will be in the correct directory.
and be able to access `conf/storage/` etc

I know that you were able to build images with this process, but how have you tested them?
have you tried running the image created by this process using `docker run`?
have you `docker exec`ed into the container and checked the expected file structure?

Reply:
So this happens because the repo cloning that used to be done here created an `e-mission-server` dir in `/usr/src/app`, and we'd need to `cd` into it to access everything. With having the Dockerfile copy files from the repo it's in instead, I just put everything into `WORKDIR /usr/src/app` directly and there is no `e-mission-server` dir to `cd` into.

Honestly, I made it like this without much thought. That's how I saw others doing it and just did the same. I don't know if it would be good to have a separate server dir, but the image exists just to be the server, so it looks a little redundant to me.

Reply:
Thank you for the clarification.